docs/src/test/java/docs/http/javadsl/HttpServerActorInteractionExample.java

```diff
@@ -96,24 +100,39 @@ private Route createRoute() {
   }
 
   static class Bids {
-    final List<Bid> bids;
+    public final List<Bid> bids;
 
     Bids(List<Bid> bids) {
       this.bids = bids;
     }
   }
 
-  //#actor-interaction
-  static class Auction extends UntypedActor {
-    static Props props() {
-      return Props.create(Auction.class);
-    }
+}
 
-    @Override
-    public void onReceive(Object message) throws Throwable {
+//#actor-interaction
```

Comment:
We could also include the actor code in the example, to do so remove this line and the same one after the actor class.